spec/dependabot/file_updaters/java_script/npm_and_yarn_spec.rb

```diff
@@ -514,7 +514,7 @@
                       "0c6b15a88bc10cd47f67a09506399dfc9ddc075d")
 
               expect(updated_yarn_lock.content).to include("is-number")
-              expect(updated_yarn_lock.content).to_not include("0c6b15a88bc")
```

Comment:
looks like this previously installed the latest version of is-number

Reply:
ah so this is empty when the version is a git path: https://github.com/dependabot/dependabot-core/blob/29be4a6bf547d737314c6418cfc0439e3e74a665/helpers/yarn/lib/replace-lockfile-declaration.js#L31

meaning `oldPackageReqs[0]` was undefined which resulted in `is-number@undefiend` which I guess is interpreted as the latest version.. 🙈

Reply:
Ah man, I always knew this test wasn't great (had avoided pointing to the correct git SHA because I knew we weren't always getting it right). Hadn't realised how wrong it was! 🙈